src/smc-util/message.js

```diff
@@ -1751,8 +1751,8 @@ API(
       }
     },
     desc: `\
-Copy a file or directory from one project to another. User must be
-owner or collaborator on both projects.
+Copy a file or directory from one project to another.
+User must be owner or collaborator on both projects.
```

Comment:
Is it really true "User must be owner or collaborator on both projects."?   I thought it would be "User must have write access to project".  For example, can admins do this?  What if we make a new type of users that is neither owner or collaborator, but it has write access?

Reply:
well, I also wondered about this, … and I only introduced a linebreak when I read it. This sentence appears several times, the earliest iteration I found appeared 2 years ago: https://github.com/sagemathinc/cocalc/blame/23e5a03d266baac42083ca8ba6a6f2f22a7b9d67/src/smc-util/message.coffee#L794

The sentence above: https://github.com/sagemathinc/cocalc/blame/99a7d1697e78f848c0e64ec4b26ed83035d0ebfe/src/smc-util/message.coffee#L1484

I'll change this to the formulation I used for the other endpoints.

Reply:
/cc @DrXyzzy  FYI we're talking about what you wrote